public/static/docs/user-guide/privacy.md

```diff
@@ -16,13 +15,16 @@ service. See
 
 ## Accessing user data
 
-DVC access Google resources from your machine you run DVC on. Your machine
+DVC accesses Google resources from the machine where you use DVC. Your machine
 communicates directly with the Google APIs.
 
-DVC project never receives your data or the permission to access your data. The
-owners of the project can only see anonymous, aggregated information about usage
-of tokens obtained through its OAuth client, such as which APIs and endpoints
-are being used.
+The _DVC_ application never receives your data or the permission to access your
```

Comment:
@MaxRis I changed "DVC project" for "_DVC_ application" but I'm not sure which one is which. Is there a GCP project called DVC? Or is it just a Gogle application that's called that? Can you share some links to all and any of these entities so I can express it here clearly? Thanks

Reply:
> Can you share some links to all and any of these entities

Maybe read access to my Google account if you think it could be useful (jorge@orpinel.com)

Reply:
@jorgeorpinel I afraid you should be invited by @shcheklein to have access to the GCP project.
So, yes, we are talking here about GCP project and `DVC` is a title for this project given through OAuth Consent Screen. `DVC` name is visible to the users, but we are explaining that GCP project behind doesn't receive access to any users' data.

Reply:
So the right term is "DVC GCP Project" right? There's no _DVC_ application? Thanks

Reply:
Just "DVC Application" would mean dvc command line application itself, I believe. Yes, "DVC GCP Project" should be fine to go with.

Reply:
DVC GCP Project sounds too complicated. The right term is DVC application I believe. This is the one we get secret/token for and the one used to authenticate (generate credentials) for users.

Reply:
in this specific case DVC project was the right phrase though. We refer to the DVC team, etc.

Reply:
So, I would keep it the way it was before probably.

Reply:
> Just "DVC Application" would mean dvc command line application itself 

Exactly. I'm trying to avoid confusions. Alright then, I changed all the instances of _DVC_ application for DVC GCP Project in . Thanks Max!

Reply:
Oops I just noticed your comments here @shcheklein 

> in this specific case DVC project was the right phrase though. We refer to the DVC team

I don't think that's the case, this refers to the dvc project or app on GCP which we own, right? See, this is the confusion I'm trying to clear up. If you could five me read access to the GCP I could figure it out on my own.

Reply:
kk. still looks like an overkill DVC GCP Project sounds way too complicated. DVC project is enough in either case. You can make _DVC project_ everywhere (but we already use _DVC_ to refer to the GCP app/project).

(Also, most likely the right term is application, not project from the Google console perspection:

<img width="488" alt="Screen Shot 2020-02-10 at 4 24 59 PM" src="https://user-images.githubusercontent.com/3659196/74202648-ece0ab00-4c21-11ea-915a-bcba23c0da57.png">
)

Reply:
> still looks like an overkill DVC GCP Project... DVC project is enough in either case

Yes, I don't disagree with that. But DVC project is a term we have in the glossary also so Idk... Unless we move to stop using that term in favor of DVC repo always – please see https://github.com/iterative/dvc.org/pull/931#discussion_r376181149 BTW, been trying to have your attention to that question haha.

> most likely the right term is application

Perfect, I'm using "_DVC_ Application" then! Done in 9e14575a

Reply:
👍